apiary.apib

```diff
@@ -361,6 +361,36 @@ We consider the following changes to be backwards-incompatible:
 - Changing the format of a parameter of API responses (eg. integer to float)
 - Making validation of a parameter more strict (eg. making it required)
 - Returning a different HTTP status code.
+
+<a name="additions"></a>
+### Additions
+
+We list all backwards-compatible additions here. These are currently available in all published versions.
+(There is also a [list of backwards-incompatible upgrades](#changelog) available, but those only apply if you [upgrade your API version](#upgrading-your-api-version).)
+
+#### Latest
```

Comment:
```suggestion
#### April 2019
```

Reply:
I don't want to do this, because if an integrator already looked at April 2019, I don't want them to have to look back at April. I want the list per month to be "immutable".